Packs/EclecticIQ/Integrations/EclecticIQIntelligenceCenterv3/EclecticIQIntelligenceCenterv3.py

```diff
@@ -2022,6 +2022,7 @@ def get_entity(eiq_api):
 
     elif query_result is False:
         return "No entities found in EclecticIQ Intelligence Center."
+    return None
```

Comment:
We can also do something like this which allows maintaining current flows in addition to complying with our code standards.
```suggestion
    return_results("No entities found in EclecticIQ Intelligence Center.")
    return None
```